locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+
+#: src/components/BackButton.js:33 src/components/PinPasswordWrapper.js:60
+#: src/screens/LoadWallet.js:119 src/screens/NewWallet.js:164
+#: src/screens/NewWallet.js:177
+msgid "Back"
+msgstr "Voltar"
+
+#: src/screens/LoadWallet.js:120
+msgid "Import data"
+msgstr "Importar wallet"
+
+#: src/screens/LoadingAddresses.js:66
+msgid "Loading transactions..."
+msgstr "Carregando transações..."
+
+#: src/screens/LoadingAddresses.js:70
+msgid "Please wait while we load the transactions of all your addresses."
+msgstr "Por favor, aguarde enquanto nós carregamos as suas transações."
+
+#: src/screens/LoadingAddresses.js:71
+msgid ""
+"You will be automatically redirected to the wallet when we finish loading "
+"them."
+msgstr "Você será automaticamente redirecionado para a wallet quando terminar."
+
+#: src/screens/LoadingAddresses.js:72
+#, javascript-format
+msgid "<strong>Addresses found:</strong> ${ this.props.addressesFound }"
+msgstr "<strong>Endereços encontrados:</strong> ${ this.props.addressesFound }"
+
+#: src/screens/LoadingAddresses.js:73
+msgid "<strong>Transactions found:</strong> ${ this.props.transactionsFound }"
+msgstr ""
+"<strong>Transações encontradas:</strong> ${ this.props.transactionsFound }"
+
+#: src/screens/LockedWallet.js:90
+msgid "Your wallet is locked. Please write down your PIN to unlock it."
+msgstr ""
+"Sua wallet está bloqueada. Por favor, entre com o seu PIN para desbloqueá-la."
+
+#: src/components/ModalResetAllData.js:67 src/screens/LockedWallet.js:96
+#: src/screens/Settings.js:125
+msgid "Reset all data"
+msgstr "Resetar todos os dados"
+
+#: src/screens/LockedWallet.js:97
+msgid "Unlock"
+msgstr "Desbloquear"
+
+#: src/screens/NewWallet.js:154
+msgid ". A new wallet is generated by 24 words."
+msgstr ". Uma nova wallet é gerada a partir de 24 palavras."
+
+#: src/screens/NewWallet.js:155
+msgid ""
+". To have access to this wallet you must have all the words saved in the "
+"same order we will show to you."
+msgstr ""
+". Para recuperar o acesso a sua wallet, você deve salvar essas 24 palavras "
+"na mesma ordem que nós iremos te mostrar."
+
+#: src/screens/NewWallet.js:156
+msgid ""
+". If someone manages to discover your words they can steal your tokens, so "
+"we advise you to save your words physically and don't show them to anyone."
+msgstr ""
+". Se alguém tiver acesso as suas palavras, seus tokens podem ser roubados. "
+"Por isso, nós recomendamos que você salve suas palavras e não as mostre pra "
+"ninguém."
+
+#: src/screens/NewWallet.js:160
+msgid "Ok, I got it!"
+msgstr "Ok, eu entendi!"
+
+#: src/screens/NewWallet.js:165
+msgid "Create my words"
+msgstr "Criar minhas palavras"
+
+#: src/screens/NewWallet.js:174
+msgid "Your words have been created!"
+msgstr "Suas palavras foram criadas com sucesso!"
+
+#: src/screens/NewWallet.js:175
+msgid ""
+"You should save them in a non-digital media, such as a piece of paper. We "
+"advise you to do it now, but you can do it later."
```

Comment:
Is it really possible to do it later? I thought we didn't keep the seed, just the private key.

Reply:
We keep the seed so we can regenerate the wallet if the user sets a passphrase. We could do it only with the private key but since we had decided we wanted to allow a late backup, we ended up saving the seed encrypted by the password.

Reply:
Exactly. Maybe we can disable it and discard the seed. But then it wouldn't be possible to change the passphrase.

Reply:
I moved the pt-br translation to PR #74. So, we can merge this one first, and discuss the best translation there.